boards/arm/efr32_radio/pre_dt_board.cmake

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) 2021 Linaro Limited
+# SPDX-License-Identifier: Apache-2.0
+
+# SPI is implemented via usart so node name isn't spi@...
+list(APPEND EXTRA_DTC_FLAGS "-Wno-spi_bus_bridge")
```

Comment:
In the tests and samples only these modified silabs boards use spi. But the issue exists for all silabs boards. SPI always uses the usart peripheral (named usart).
While this change here is sufficient for the ci, should the same file `pre_dt_board.cmake` be added for all other silabs boards here or after v2.5.0 is released in a separate pr?

Reply:
Lets do it post v2.5.0 as a separate PR.